src/components/CioPlpBrowse/reducer.ts

```diff
@@ -0,0 +1,114 @@
+import {
```

Comment:
This is the same reducer as CioPlpGrid so it contains logic for both Search/Browse. Depending on what we decide for the other comments, I think we need to whether or not to keep everything separate or all in one.

As it is I don't think we need this separate file, we can use the same reducer from CioPlpGrid.

Reply:
I think the only used parts of this file are the types. `useBrowseResults` seems to be using the reducer from `CioPlpGrid`